addons/sale/views/sale_portal_templates.xml

```diff
@@ -261,29 +255,30 @@
                                     </div>
                                     <p>
                                         <!-- The widget associated with this modal will hide and show divs in function of the amount selected. -->
-                                        <div id="o_sale_portal_use_amount_total">
-                                            By paying this proposal, I agree to the following terms:
-                                        </div>
-                                        <div t-if="prepayment_available"
+                                        <span id="o_sale_portal_use_amount_total">
```

Comment:
you need to wrap it in `if` as well I think. Because here is how it looks like.
![image](https://github.com/odoo/odoo/assets/127864697/0399966e-d1c9-4ca9-a1f6-33a1b50f055d)